### ISSUE TITLE
Broken local links no error raised

### ISSUE THREAD

**Issue by Alex2357** [DavidAnson/vscode-markdownlint#61]
Hi,

I would like to have check for the local links to verify if they're valid. Right now I just added in my file

```
[see also main doco](/a/valid/path/a-not-existing-markdown-file.md)
```

and no error is reported.

**Comment by DavidAnson** [comment]
This should eventually be covered by the following issue in the underlying `markdownlint` project: https://github.com/DavidAnson/markdownlint/issues/121

**Comment by Alex2357** [comment]
> This should eventually be covered by the following issue in the underlying `markdownlint` project: [DavidAnson/markdownlint#121](https://github.com/DavidAnson/markdownlint/issues/121)

Thanks for the quick response
